### DISCUSSION TITLE
- Homework 3/Final Project - Katie Popiela

### DISCUSSION THREAD

**Discussion by giscus** [DACSS/DACSS_601_August2022_v2#258] [Announcements]
# - Homework 3/Final Project - Katie Popiela

https://dacss.github.io/DACSS_601_August2022_v2/posts/Homework3(2)_KatiePopiela.html

**Comment by LarriMill** [comment]
Hi Katie, you do a phenomenal job contextualizing your questions and work. I am SO excited to see what you do with this project in the future and I encourage you to continue this work. You also have great reflections on what works and what needs improved.

**Reply by mrolfe** [reply]
Seconding these comments. I think that filtering so heavily sort of detracts some from your ability to make critical points (e.g., understanding the relationship between ethnic fractionalization and democratization depends on understanding non-authoritarian countries also) But I love the topic, and think that even with some of these limited data sets, you could effectively use graphics and data exploration to get to some of the interesting points.

**Comment by LarriMill** [comment]
Hi Saaradhaa, this is EXCELLENT. You do an absolutely lovely job describing your questions, the context of your data, what you are doing, and what you fine. Wonderful work.

**Comment by LarriMill** [comment]
Hi Miranda,
You do a really fantastic job of explaining to your reader what you are doing. It is easy to follow your work which is really important in this type of write-up. Your questions are clear and easy to follow. I also appreciate that you explained your issues with your dataset and kept working.

**Comment by LarriMill** [comment]
Hi Lindsay, wow, your graphs are amazing! Really clear, fascinating work. Well done.

**Comment by LarriMill** [comment]
Hi Young Soo, this is a really fascinating, well thought out project! I love the clarity of your intentions, the theories, and the external knowledge that you used. My advice is to make sure that your audience has enough information to understand what you are referencing (for example, I didn't know beforehand what a Philip's curve should look like), and it would be good to have more variety in your graphs. Well done overall.

**Comment by LarriMill** [comment]
Hi Tyler, this is written well, with beautiful graphs and clear questions. I would have liked to see you explain the dataframe summaries (or not include them), and it would have been nice to see a more in depth analysis. What you have is great, there just needs to be more.
